src/govuk/settings/settings.test.js

```diff
@@ -39,7 +39,7 @@ describe('The settings layer', () => {
 
   describe('Sass documentation', () => {
     it('associates everything with a "settings" group', async () => {
-      const docs = await sassdoc.parse(join(configPaths.src, 'settings', '*.scss'))
+      const docs = await sassdoc.parse(join(configPaths.src, 'govuk/settings/**/*.scss'))
```

Comment:
You'll still see us using `/` forward slashes within `join()` as @romaricpascal pointed out that `normalize()` is also run behind the scenes 🙌 

https://nodejs.org/api/path.html#pathjoinpaths
>The `path.join()` method joins all given path segments together using the platform-specific separator as a delimiter, then **normalizes the resulting path**.